src/ducks/payments/service.js

```diff
@@ -0,0 +1,75 @@
+import logger from 'cozy-logger'
+import { biApi, getPayment } from './api'
+import { makePayload } from './helpers'
+import { waitForRealtimeEvent } from 'cozy-harvest-lib/dist/services/jobUtils'
+import { TRANSACTION_DOCTYPE } from 'doctypes'
+import get from 'lodash/get'
+
+const log = logger.namespace('payment-service')
+
+export const createBiPayment = async client => {
+  const jobResponse = await client.stackClient.jobs.create('konnector', {
+    mode: 'getPaymentToken',
+    konnector: 'caissedepargne1'
+  })
+  const event = await waitForRealtimeEvent(
+    client,
+    jobResponse.data.attributes,
+    'result',
+    30 * 1000
+  )
+  return event.data.result
+}
+
+const savePaymentCreation = async (client, paymentCreation) => {
+  const label = get(paymentCreation, 'instructions[0].label')
```

Comment:
Nit: Seems like intrustructions[0] is used everywhere, it could be put into a variable.